src/languages/pt_BR.json

```diff
@@ -0,0 +1,271 @@
+    "captainName": "Nome do Capitão",
+    "randomizeName": "Randomizar nome"
+  },
+  "rolemodal": {
+    "header": "Deina teu cargo",
+    "draftUrlCallToAction": "Envie esta url para o outro capitão:",
+    "spectateUrlCallToAction": "Envie esta url para os espectadores:",
+    "copyLabel": "Copie",
+    "copiedLabel": "Copiado!",
+    "callToAction": "Defina teu cargo para este draft:",
+    "role": {
+      "spectator": "Espectador",
+      "host": "Anfitrião",
+      "guest": "Convidado"
+    }
+  },
+  "HOST": "Capitão Anfitrião",
+  "GUEST": "Capitão Convidado",
+  "you": "Você",
+  "Picks": "Escolhas",
+  "Bans": "Banimentos",
+  "Start": "Início",
+  "End": "Fim",
+  "<0></0>PICK": "<0></0>ESCOLHER",
+  "<0></0>BAN": "<0></0>BANIR",
+  "flip": "Inverta a posição do Anfitrião e do Convidado",
+  "smooch": "Alternar modo smooch",
+  "validationFailed": "A(s) validação(ões) falhou(aram):",
+  "errors": {
+    "VLD_000": "O draft não está esperando por ações atualmente",
+    "VLD_001": "O usuário ativo não deve estar agindo de acordo com a predefinição",
+    "VLD_002": "Ação inesperada de acordo com a predefinição",
+    "VLD_010": "Ação não permitida com essa civilização",
+    "VLD_900": "O servidor não conseguiu processar a predefinição corretamente",
+    "VLD_901": "A predefinição contém uma sequência inválida de turnos paralelos",
+    "VLD_902": "A predefinição contém um turno paralelo que não requer interação do jogador",
+    "VLD_903": "A predefinição contém um turno paralelo seguido por um turno que não requer interação do jogador",
+    "VLD_904": "O último turno da predefinição é um turno paralelo, o que não é possível",
+    "VLD_905": "A predefinição contém um turno paralelo que deve ser executao pelo mesmo jogado do próximo turno",
+    "VLD_906": "A predefinição contém um turno com banimento oculto e uma escolha sem revelação no meio",
+    "VLD_907": "A predefinição contém um turno oculto que nunca é revelado",
+    "VLD_908": "A prredefinição não contém turnos",
+    "VLD_909": "O presetId fornecido é inválido"
+  },
+  "menu": {
+    "welcome": "Bem-vindo",
+    "hostOrJoin": "Seja Anfitrião ou Entre",
+    "spectate": "Observar",
+    "practice": "Pratique",
+    "howItWorks": "Como isso funciona"
+  },
+  "instructions": {
+    "home": {
+      "title": "O que é o Captains Mode?",
+      "1": "Captains mode é um seletor de civilizações baseado em turnos.",
+      "2": "O Administrador do torneio cria uma <1>Definição</1> que pode ser usada para criar <3>Drafts</3>. Durante um Draft, os dois capitães escolhem, banem, roubam, ou excluem civilizações em uma ordem prédefinida conforme as regras na Definição escolhida para determinar quais civilizações estarão disponíveis para cada capitão em uma partida.",
+      "3": "Para cada turno, os capitães têm <1>30 segundos</1>. Caso o tempo acabe, uma civilização aleatória será selecionada ou banida para o capitão."
+    },
```

Comment:
This closing bracket is wrong here; `getStarted` and `reportIssue` still belong under `home`

Reply:
Good catch. I should have QA-ed properly 🤦🏼